src/main/java/net/fabricmc/loom/util/srg/ForgeMappingsMerger.java

```diff
@@ -326,8 +330,8 @@ private void resolveConflicts() {
 		List<MethodData> hasTiny = CollectionUtil.filter(methods, MethodData::hasTiny);
 
 		// Record conflicts if needed
-		if (hasTiny.size() > 1) { // Multiple methods map to this SRG name
-			// Sometimes unrelated methods share an SRG name. Probably overrides from a past version?
+		if (hasTiny.size() > 1) { // Multiple methods map to this new name
+			// Sometimes unrelated methods share an new name. Probably overrides from a past version?
```

Comment:
```suggestion
			// Sometimes unrelated methods share a SRG name. Probably overrides from a past version?
```
This is specific to srg, Mojang mappings will obv have duplicate names like this